src/Microsoft.AspNet.OData.Shared/EdmPatchMethodHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.OData.Edm;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.OData
+{    
+   
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update
```

Comment:
>Handler Class to handle users methods for create, delete and update [](http://example.com/codeflow?start=8&length=67)

Add that this is for the case where there is no CLR type -- PatchMethodHandler is used when there is a clr type.